databases/sql_snippets_postgres.rst

```diff
@@ -4,52 +4,39 @@ SQL Snippets (Postgres)
 Show Queries Running in Postgres
 --------------------------------
 
-show all queries:
+Show all Queries
```

Comment:
Im not sure how it's done in general but but lots of words in titles are written uppercase even if they normally would be written lowercase (Queries L4&L7, Running L4).